e2e/wdio/core/tests/alert.e2e-spec.ts

```diff
@@ -0,0 +1,90 @@
+import { AlertPo } from '../pages/alert.po';
+import {
+    click,
+    doesItExist,
+    getElementArrayLength,
+    getImageTagBrowserPlatform,
+    refreshPage,
+    saveElementScreenshot,
+    waitForElDisplayed,
+    waitForInvisibilityOf,
+    checkElementScreenshot, scrollIntoView
+} from '../../driver/wdio';
+import { checkElArrIsClickable } from '../../helper/assertion-helper';
+
+describe('alert test suite', function() {
+    const alertPage = new AlertPo();
+    const {
+        alerts,
+        closeAlertButton,
+        openOverlayButton,
+        popupAlert,
+        button,
+        openCustomAlertButton
+    } = alertPage;
+
+    beforeAll(() => {
+        alertPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForElDisplayed(alerts);
+    }, 1);
+
+    describe('main checks', function() {
+        it('should check ability to dismiss alert', () => {
+            checkElArrIsClickable(closeAlertButton);
```

Comment:
That the button is clickable doesn’t prove that the user is able to dismiss an alert

Reply:
added better check, but hard to show that alert gone due to the component's implementation. Selectors still found after dismissed, but not displayed.